src/BackgroundCells.js

```diff
@@ -91,6 +91,27 @@ class BackgroundCells extends React.Component {
       longPressThreshold: this.props.longPressThreshold,
     })
 
+    let selectorClicksHandler = (point, actionType) => {
+      if (!isEvent(findDOMNode(this), point)) {
+        let rowBox = getBoundsForNode(node)
+        let { range, rtl } = this.props;
+
+        if (pointInBox(rowBox, point)) {
+          let width = slotWidth(getBoundsForNode(node),  range.length);
+          let currentCell = getCellAtX(rowBox, point.x, width, rtl, range.length);
+
+          this._selectSlot({
+            startIdx: currentCell,
+            endIdx: currentCell,
+            action: actionType,
+          })
+        }
+      }
+
+      this._initial = {}
+      this.setState({ selecting: false })
+    };
```

Comment:
Refactor handler functions, as they are absolutely the same for `click` and `doubleClick` events, except `action` property.